### ISSUE TITLE
Bulk import plugin: Add missing assertion (expect) to runAccessibilityTests

### ISSUE THREAD

**Issue by christoph-jerolimov** [redhat-developer/rhdh-plugins#3579]
The bulk import plugins has playwright tests that include a bunch of accessibility assertions.

The tests runs `runAccessibilityTests` from `@workspaces/bulk-import/e2e-tests/utils/helpers.ts`, but the `accessibilityScanResults.violations` isn't used in any assertion.

Add `expect(accessibilityScanResults.violations, 'Accessibility violations found').toEqual([]);` to that function.
